adapters/lora-vertx/pom.xml

```diff
@@ -60,6 +60,12 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.1.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
no need for this, the parent POM already declares the jupiter dependency. Please remove, as it also overrides a *managed dependency* ...

Reply:
ok, removed